packages/cli/test/agent-customisation.test.js

```diff
@@ -0,0 +1,312 @@
+import { afterEach, beforeEach, describe, expect, it } from 'vitest';
```

Comment:
Overall we may want to create the dummy plugins / providers / actions beforehand and have them as files that we just read in that way we abstract away the writing of a file for each test case.